src/main/webapp/style.css

```diff
@@ -88,8 +86,24 @@ html {
   width: 70%;
 }
 
-.navbar {
-  background-color: #f3f3f3;
+#add-btn-cont {
+  margin: 30px auto auto auto;
+  width: 200px;
+}
+
+#add-btn-cont .btn,
+#submit-btn-cont .btn {
+  background-color: #4c8bf5 !important;
+}
+
+#add-btn-cont .btn,
+#add-btn-cont .btn:hover {
+  width: 200px !important;
+}
+
+#add-btn-cont .btn:hover,
```

Comment:
Is it supposed to be formatted this way? Can you look into this from the style guide?

Reply:
Yeah the `selector` continues until the comma, which should all be on one line

Reply:
Can you add Sahil to this review. I feel he would be better for this stuff.